test/TCK/TCK.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;net45</TargetFrameworks>
```

Comment:
Maybe we need to update the Readme.md file ;) because we have references to DotNetCore 1.0 🙂 

Can you check?

Reply:
Hi!

Sure! I thought it would be better to do it in a separate PR, but the info related to dependencies can be updated now.

Thanks!